docker/manta.Dockerfile

```diff
@@ -35,52 +22,20 @@ RUN apt-get install -yq libssl-dev
 
 RUN mkdir -p /usr/local/bin
 RUN mkdir -p /usr/share
-RUN mkdir -p $SUBSTRATE_BASE_PATH
 
 ADD $PARA_BINARY_URL $PARA_BINARY_PATH
 RUN chmod +x $PARA_BINARY_PATH
 RUN ldd $PARA_BINARY_PATH
 RUN $PARA_BINARY_PATH --version
 
+# Get manta and kusama genesis file
 ADD $PARA_GENESIS_URL $PARA_GENESIS_PATH
 ADD $RELAY_GENESIS_URL $RELAY_GENESIS_PATH
 
-EXPOSE $SUBSTRATE_PORT
-EXPOSE $SUBSTRATE_RPC_PORT
-EXPOSE $SUBSTRATE_WS_PORT
-
-ENV PARA_BINARY_PATH=$PARA_BINARY_PATH
-ENV PARA_GENESIS_PATH=$PARA_GENESIS_PATH
-ENV SUBSTRATE_BASE_PATH=$SUBSTRATE_BASE_PATH
-ENV SUBSTRATE_PORT=$SUBSTRATE_PORT
-ENV SUBSTRATE_RPC_PORT=$SUBSTRATE_RPC_PORT
-ENV SUBSTRATE_WS_PORT=$SUBSTRATE_WS_PORT
-ENV SUBSTRATE_RPC_CORS=$SUBSTRATE_RPC_CORS
```

Comment:
not sure its a good idea to get rid of all these ENV definitions.
@grenade might have an opinion on / a need for these and community collators can be using them to customize their containers

OTOH i guess people can always overwrite the entrypoint if they need to customize so this is probably ok